.github/workflows/e2e.yaml

```diff
@@ -15,13 +15,14 @@ jobs:
     runs-on: kvm-host
     env:
       TIMEOUT_SCALE: 2
-      CLUSTER_NAME: cluster-k3s
+      CLUSTER_NAME: cluster-rke2
       CLUSTER_NS: fleet-default
-      INSTALL_K3S_VERSION: v1.23.9+k3s1
+      K8S_VERSION: v1.23.8+rke2r1
+      INSTALL_K3S_VERSION: v1.23.8+k3s2
```

Comment:
_No need to change something, just a remark here._

I understand why we have `K8S_VERSION` and `INSTALL_K3S_VERSION` but I find it a little bit confusing.
Maybe variable names can be improved, at the end, `INSTALL_K3S_VERSION` is the cluster where Rancher manager is installed and `K8S_VERSION` is the version for the elemental nodes.

Reply:
`INSTALL_K3S_VERSION` cannot be changed as this variable is used by the K3s installer. But I will rename `K8S_VERSION` and also add comments to explain these variables.